config/esp32/components/chip/Kconfig

```diff
@@ -712,7 +712,7 @@ menu "CHIP Device Layer"
                 should not start advertising automatically after power-up.
 
         config USE_BLE_ONLY_FOR_COMMISSIONING
-            depends on BT_ENABLED
+            depends on ENABLE_CHIPOBLE
```

Comment:
I assume this option dependent on BT_ENABLED?

Reply:
Yes it does.